### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by mmarcotte**
As we head towards production, ready these two simple templates for github users to use as the basis for creating new issues with the repository.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
```
Comment (julialeague) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: What would this be?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+labels: ''
```
Comment (julialeague) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: Would this be like our label tag on the issue?

**Review by julialeague — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (mmarcotte) [reply .github/ISSUE_TEMPLATE/feature_request.md:4]: This is just the default setting. So when you create a new issue with this template, the default title will be blank. This setting lets you specify the default, perhaps a prefix like the BUG one.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (mmarcotte) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: Yep. You have the option to specify labels by default. I figured these would most likely be added by me after grooming them.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (julialeague) [reply .github/ISSUE_TEMPLATE/feature_request.md:4]: Oh, does this interact with the GitHub API somehow?

Comment (julialeague) [reply .github/ISSUE_TEMPLATE/feature_request.md:4]: I answered my own question and read the docs for issue templates. Never mind, I get it now. 😆

**Pull request closed without merging**